Makefile

```diff
@@ -53,6 +57,10 @@ dependencies-npm:
 build: ## Build the Docker image for the OTP release
 	docker build --build-arg APP_NAME=$(APP_NAME) --build-arg APP_VERSION=$(APP_VERSION) --rm --tag $(APP_NAME):$(DOCKER_IMAGE_TAG) .
 
+.PHONY: push
+push: ## Push the Docker image
+	docker push $(DOCKER_REGISTRY)/$(APP_NAME):$(DOCKER_IMAGE_TAG)
```

Comment:
In order to push to `DOCKER_REGISTRY`, I think you first need to tag the image with `$(DOCKER_REGISTRY)/$(APP_NAME):$(DOCKER_IMAGE_TAG)`

Reply:
You’re right! For example if I want to push it as `mirego/elixir_boilerplate`, I have to : 

```
$ docker tag elixir_boilerplate:latest mirego/elixir_boilerplate:latest
$ DOCKER_REGISTRY=mirego make push
```

What do you suggest we do? Should `DOCKER_REGISTRY=mirego make push` tag **and** push the image?

Reply:
Or should we change the `build` target to this?

```bash
docker build --build-arg APP_NAME=$(APP_NAME) --build-arg APP_VERSION=$(APP_VERSION) --rm --tag $(DOCKER_REGISTRY)/$(APP_NAME):$(DOCKER_IMAGE_TAG) .
```

Reply:
I think building an image and and pushing to a repo are 2 separate concerns! One might want to build the image locally to test the OTP release, but the CI process publish the images to the repo…

Changing the `push` target to do both `tag` and `push` with the `DOCKER_REGISTRY` url would make sense _imho_!

Reply:
Check it again! 😄